SourceX/controls/controller.h

```diff
@@ -16,11 +16,11 @@ struct ControllerButtonEvent {
 class Controller {
 public:
 	// Raw axis values.
-	float leftStickX, leftStickY, rightStickX, rightStickY;
+	float leftStickX = 0, leftStickY = 0, rightStickX = 0, rightStickY = 0;

```

Comment:
Can we declare each field in its own line?

Reply:
Sure!